src/providers/execution/contracts/cs_accounting.py

```diff
@@ -26,6 +28,7 @@ def fee_distributor(self, block_identifier: BlockIdentifier = "latest") -> Check
         )
         return Web3.to_checksum_address(resp)
 
+    @lru_cache
```

Comment:
Not sure that it make sense here. We need to cache CSParams values by curve_id only

Reply:
We have only a node operator ID at hand, so we go: `no_id` -> `curve_id` -> `params`.
An operator has multiple validators, hence it will be queried multiple times, so we cache `no_id` -> `curve_id` part. Then we cache the `curve_id` -> `params` in `get_strikes_params` at the contract level. And, in csm extention `get_strikes_params` we have a redundant cache.